white-paper/wp.tex

```diff
@@ -683,51 +683,49 @@ \subsection{Features}\label{sec:features}
     with immediate consistency.
   \item[Durability]
     The system must have enough replicas to recover itself in case of corruption.
-    Corrupted repository could be responsible for recovering itself using replica nodes.
-  \item[Self management (rename?)]
+    The corrupted repository could be responsible for recovering itself using replica nodes.
+  \item[Self-management (rename?)]
     Each node performs cleanup when needed (\code{git gc}) and may remove replica
-    from storage on read inactivity.
-    A node should be able to find and synchronize new repository on read,
-    after that it should be up to date on new updates.
+    from storage on reading inactivity.
+    A node should be able to find and synchronize new repository on reading. After that, it should be up to date on new updates.
   \item[Maintainability]
     Node administrator can change the storage, and perform data migration from one storage
     to another.
-    Repository administrators are able to add or delete node for new region and
-    get all nodes status for repository.
+    Repository administrators can add or delete a node for the new region and
+    get status of all nodes in the repository.
   \item[Auditability]
-    Node doesn't perform access control operations, but logs all
+    Node doesn't perform access control operations but logs all
     requests with identity and performed operation.
   \item[Analytics]
     Node collects statistics for each repository and usage metrics, such as
-    push and pull operations, etc. The system keeps the whole statistics about
-    nodes, e.g.\@ how many nodes contains each repository, the state of nodes, etc.
+    Push and Pull operations, etc. The system keeps the whole statistics about
+    nodes, e.g.\@ how many nodes are contained in each repository, the state of nodes, etc.
 \end{description}
 
 \subsection{Load balancing}\label{sec:load-balancing}
 
 The front-end decides how to redirect incoming fetch requests to replica nodes.
 It knows network addresses and current availability of repository replicas,
 so it can decide how to route next fetch request to git repository back-end storage node.
-On read access to repository, it sends a health check message to replica nodes to be sure
+On read access to the repository, it sends a health check message to replica nodes to verify
 which are available. It stores the result in local cache with configurable time to live
-properties. When front-end unable to access some of back-end node, it marks it as unavailable
-and forget about it for some time. Then in check availability periodically.
-It uses ``Round robin'' load balancing for fetch requests, since it provides uniformly distributed
+properties. When the front-end cannot access some of the back-end nodes, it marks it as unavailable
+and forgets about it for some time. Then in check availability periodically.
+It uses "Round robin" load balancing for fetch requests since it provides a uniformly distributed
```

Comment:
@vtchem please use `` + '' (double ` and double ') as LaTeX quotes, see https://tex.stackexchange.com/questions/531/what-is-the-best-way-to-use-quotation-mark-glyphs#10677